test/components/entity/filters.spec.js

```diff
@@ -30,37 +30,43 @@ describe('EntityFilters', () => {
     describe('initial request', () => {
       it('does not filter by conflict by default', () =>
         load('/projects/1/entity-lists/trees/entities', { root: false })
-          .beforeEachResponse((_, { url }, index) => {
-            if (index !== 1) return;
-            const filter = relativeUrl(url).searchParams.get('$filter');
-            filter.should.not.match(/__system\/conflict ne null/);
-          }));
+          .testRequestsInclude([{
+            url: ({ searchParams }) => {
+              searchParams.get('$top').should.equal('250');
+              const filter = searchParams.get('$filter');
+              filter.should.not.match(/__system\/conflict ne null/);
+            }
+          }]));
```

Comment:
Instead of using `testRequestsInclude()`, I could have simply updated the index. However, I think that assertions based on the request index are inherently fragile. I thought I'd take the opportunity to replace it with a test pattern that's a little more resilient.